### PR TITLE
Logs in game screen

### LINKED ISSUE #567: [FEATURE] Add logs in screen
## 🚀 Feature

Display the logs in screen for better debugging in mobile and desktop

### PR DESCRIPTION
**Pull request by BertovDev**
This PR allows you to visualize console logs in screen , useful in mobile builds.
Also implements custom debug.log for only printing things when you desire ( touching a button in panel settings )

- `CustomDebugLog.CustomDebug(object message)` only works when activating `use Custom logs` in settings panel
- Show logs screen in `Show logs` button in settings panel

closes #567 

- [x] Tested in Unity Desktop
- [x] Tested in Android

### CODE REVIEW

**Comment by tvillegas98** [comment]
Seems to be working well! Can we add the same button for every screen? it will be helpful

**Comment by BertovDev** [comment]
> Seems to be working well! Can we add the same button for every screen? it will be helpful

Absolutely!!

**Comment by manucamejo** [comment]
@BertovDev 

Why do the buttons appear in the lobbies scene?
image

I can't scroll through the logs and Muflus is naked
image

File: client/Assets/Scripts/CustomLogs.cs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    }
+
+    void OnDisable()
+    {
+        Application.logMessageReceivedThreaded -= HandleLog;
+    }
+
+    void Update()
+    {
+        CustomDebugLog.CustomDebug("CUSTOMM!!");
```
Comment (ncontinanza) [inline client/Assets/Scripts/CustomLogs.cs:32]: I guess we should remove this line

**Review by ncontinanza — CHANGES_REQUESTED** [review]

**Review by ncontinanza — APPROVED** [review]

**Review by tvillegas98 — APPROVED** [review]
Nice work 😎

**Pull request merged**